x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/plan/logical/KeyedFilter.java

```diff
@@ -29,22 +29,31 @@ public class KeyedFilter extends UnaryPlan {
     private final List<? extends NamedExpression> keys;
     private final Attribute timestamp;
     private final Attribute tiebreaker;
-
-    public KeyedFilter(Source source, LogicalPlan child, List<? extends NamedExpression> keys, Attribute timestamp, Attribute tiebreaker) {
+    private final boolean missingEvent;
```

Comment:
I would rename this to be a bit more descriptive and a bit more accurate: `isMissingEventFilter`.